Shaded jackson issue with objectMapper.findAndRegisterModules()

My app uses jackson and:
```
    ObjectMapper objectMapper = new ObjectMapper(jsonFactory);
    objectMapper.findAndRegisterModules();
```
When using testcontainers-java:1.3.1, this fails with the error:
```
Caused by: java.util.ServiceConfigurationError: com.fasterxml.jackson.databind.Module: Provider org.testcontainers.shaded.com.fasterxml.jackson.module.jaxb.JaxbAnnotationModule not a subtype
	at java.util.ServiceLoader.fail(ServiceLoader.java:239)
	at java.util.ServiceLoader.access$300(ServiceLoader.java:185)
	at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:376)
	at java.util.ServiceLoader$LazyIterator.next(ServiceLoader.java:404)
	at java.util.ServiceLoader$1.next(ServiceLoader.java:480)
	at com.fasterxml.jackson.databind.ObjectMapper.findModules(ObjectMapper.java:929)
	at com.fasterxml.jackson.databind.ObjectMapper.findModules(ObjectMapper.java:912)
	at com.fasterxml.jackson.databind.ObjectMapper.findAndRegisterModules(ObjectMapper.java:948)
```

Indeed in this jar META-INF/services contains one provider conf file named   `com.fasterxml.jackson.databind.Module`but contains shaded impl: `org.testcontainers.shaded.com.fasterxml.jackson.module.jaxb.JaxbAnnotationModule`. Not sure if it's on purpose, as others are properly named.
Renaming the file `org.testcontainers.shaded.com.fasterxml.jackson.databind.Module` fixes the issue.

--- a/core/pom.xml
+++ b/core/pom.xml
@@ -136,7 +136,7 @@
         <plugins>
             <plugin>
                 <artifactId>maven-shade-plugin</artifactId>
-                <version>2.4.3</version>
+                <version>3.0.0</version>
                 <executions>
                     <execution>
                         <phase>package</phase>

